app/models/section.rb

```diff
@@ -31,6 +31,8 @@ class Section < ActiveRecord::Base
   include ActsAsSortable
   include VersionableModel
 
+  # Sort order: Number ASC
+  default_scope { order(number: :asc) }
```

Comment:
👍 I think this makes a lot of sense but we may want to highlight the change for people doing UAT in case the ordering of sections is off anywhere when customizing or using the drag-drop feature